tests/js/launcher-tests.js

```diff
@@ -0,0 +1,113 @@
+// Tests that launch the launcher with various options, while collecting istanbul code coverage reporting.
+/* eslint-env node */
+"use strict";
+var fluid             = require("infusion");
+var gpii              = fluid.registerNamespace("gpii");
+var path              = require("path");
+var jqUnit            = require("node-jqunit");
+var os                = require("os");
+var child_process     = require("child_process");
+var istanbulPath      = fluid.module.resolvePath("%gpii-launcher/node_modules/istanbul/lib/cli.js");
+var launcherPath      = fluid.module.resolvePath("%gpii-launcher/tests/js/lib/harness.js");
+var coverageOutputDir = fluid.module.resolvePath("%gpii-launcher/coverage");
+var optionsFile       = fluid.module.resolvePath("%gpii-launcher/tests/data/workerCustomOptions.json");
+
+fluid.registerNamespace("gpii.tests.launcher");
+
+gpii.tests.launcher.runSingleTest = function (that, testDef) {
+    jqUnit.test(testDef.message, function () {
```

Comment:
jqUnit.test immediately followed by jqUnit.stop is more idiomatically jqUnit.asyncTest